setup.py

```diff
@@ -3,7 +3,7 @@
 
 from setuptools import find_packages, setup
 
-__version__ = '0.0.1'
+__version__ = '0.0.2'
```

Comment:
could you copy and modify the github action from other repos so that we could do auto released?

Reply:
Sure. Thanks